store/searchlayer/user_layer.go

```diff
@@ -185,7 +178,7 @@ func (s *SearchUserStore) getListOfAllowedChannelsForTeam(teamId string, viewRes
 	if len(viewRestrictions.Channels) > 0 {
 		channels, err := s.rootStore.Channel().GetChannelsByIds(viewRestrictions.Channels, false)
 		if err != nil {
-			return nil, model.NewAppError("getListOfAllowedChannelsForTeam", "app.channel.get_channels_by_ids.app_error", nil, err.Error(), http.StatusInternalServerError)
+			return nil, errors.Wrapf(err, "Failed to get channels by ids")
```

Comment:
It is not required to use `errors.Wrapf` as you are not passing additional values, `errors.Wrap` is enough :)

Additionally, by convention go errors are in lowercase [(here)](https://github.com/golang/go/wiki/CodeReviewComments#error-strings). I know we have several places with capitalized errors but you can have your PR following best practices and conventions of go

Reply:
I changed `errors.Wrapf(...)` to `errors.Wrap(...)` and changed the error messages to lowercase.